src/utils/storage.js

```
@@ -7,7 +7,7 @@ const { readFile, writeFile } = promisify.methods(require('fs'), ['readFile', 'w
 const mkdirp = promisify(require('mkdirp'));
 
 // POSIX-only, but no one's using this on Windows yet.
-const STORAGE_ROOT_DIR = '/usr/local/var/custody';
+const STORAGE_ROOT_DIR = process.env.CUSTODY_PROC_DIR || '/usr/local/var/custody';
```

Curious is it a requirement that Linux users always set this or only if they use custody-probe?

---

Depending on distro, Linux users may need to set this. Currently, if the `/usr/local/var` dir doesn't exist (which is usually the case), the process will just crash.

---

Ah ok. I was wondering if we should throw our own error if this is not set on Linux, to help users identify they needed to set it, but I guess we can't make that a general rule.

---

I'll document here too - that should help a little.